<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmButtonGroup</class>
 <widget class="QWidget" name="frmButtonGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>467</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="ButtonGroup" name="buttonGroup4" native="true"/>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="ButtonGroup" name="buttonGroup2" native="true"/>
   </item>
   <item row="2" column="0">
    <widget class="ButtonGroup" name="buttonGroup3" native="true"/>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="ButtonGroup" name="buttonGroup1" native="true"/>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ButtonGroup</class>
   <extends>QWidget</extends>
   <header>buttongroup.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
